packages/client/src/components/SafeSettings/components/EditThreshold/EditThresholdForm.js

```diff
@@ -65,21 +63,18 @@ const EditThresholdForm = ({
           </div>
         </div>
         <div className="flex-1 is-flex is-flex-direction-column mt-5">
-          <div className="is-flex is-justify-content-space-between mb-2">
-            <label className="has-text-black">Security Strength</label>
-            <label className="has-text-black">{progress}%</label>
-          </div>
-          <div className="is-flex flex-1">
-            <div
-              className="is-flex column is-full border-light rounded-sm"
-              style={{ minHeight: 48 }}
-            >
-              <ProgressBar progress={progress} />
-            </div>
+          <div className="is-flex mb-2">
+            <label className="has-text-grey">Security Strength:</label>
+            <SecurityStrengthLabel 
+              progress={progress}
+              style={{ position: "relative", top: -3 }}
```

Comment:
```suggestion
```

Reply:
I think this styling is necessary to vertically align the label properly.  Is there a better way?

Reply:
oh never mind i see its in the next suggestion!